tests/gumdrop-test.ts

```diff
@@ -0,0 +1,390 @@
+import { PublicKey, Connection, Keypair, Transaction, SystemProgram, TransactionInstruction } from '@solana/web3.js';
+import * as anchor from "@project-serum/anchor";
+import NodeWallet from '@project-serum/anchor/dist/cjs/nodewallet';
+import { Gumdrop } from "../target/types/gumdrop";
+import { MerkleTree } from './gumdropTree';
+import { BinaryWriter } from 'borsh'
+
+async function delay(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function succeedOrThrow(txId: string, connection: Connection) {
```

Comment:
This is great. Maybe we can move it to `./utils.ts` to be reused?

Reply:
fixed